src/app/Fake.BuildServer.TeamCity/TeamCity.fs

```diff
@@ -412,9 +377,6 @@ module TeamCity =
                     match description with
                     | Some d -> TeamCityWriter.sendOpenBlock tag.Name (sprintf "%s: %s" tag.Type d)
                     | _ -> TeamCityWriter.sendOpenBlock tag.Name tag.Type
-                | TraceData.CloseTag (tag, _, TagStatus.Failed) ->
```

Comment:
So we don't need CloseTag?

Reply:
This case for CloseTag with a status of Failed is exactly the same as the CloseTag below with all TagStatus.